### ISSUE TITLE
2016-06-30

### ISSUE THREAD

**Issue by knighttime** [IALSA/time-missing-patterns#3]
@andkov, @knighttime

# Thursday June 30th, 10 am

**Meeting Agenda:**

1. simplify the patterns in the table
2. code to count the transitions from no stroke to stroke 
3. make sure each column represents a wave

**Comment by andkov** [comment]
To do:
- rename repo into "longitudinal-response-pattern"

**Comment by knighttime** [comment]
This session we: 
1. compile an R script that would prepare the raw data into the form that could be passed to the longitudinal-response-pattern function from raw files sent from RADC. (ds_raw.rds & transition-count.R)
2. Created an example file for long to wide examples. (long-to-wide-demos.R)

Next session:
continue to recode the count of transitions from no stroke to stroke

**Comment by andkov** [comment]
Something unsettling is happing in data. Some individuals have multiple entries per wave:

```r
ds_long %>% dplyr::filter(id == 521316)
      id fu_year stroke_cum
1 521316       0          1
2 521316       0          1
```

This needs to be investigated on the level of MAP curator. 

NOTE for the future: insist on using the data from the curators, otherwise chaos ensues.

**Comment by andkov** [comment]
> -	Is each data point a possible wave that exists in the study (i.e., a very small # of people (around 9) have a lot of waves, but the rest have a much smaller number…. Or, actually, those 9 just seem to have waves that are spread over a much longer period of time, though may not actually have more waves of data…?)? In other words, does each column represent a generalized wave or the waves at which each particular participant was actually enrolled in the study?

Yes. Each column represents a consecutive wave. A dot represents missingness on the measurement of interest (in this case, stroke).

**Comment by knighttime** [comment]
@andkov 
If there are duplicate entries in the data it is most likely from an incorrect merge.
When I was merging MAP I tried many times to get a correct merge and there were a few versions where I had duplication issues similar to what you are seeing.  

It may be more worthwhile to investigate why you were unable to replicate the merge code that I have in my ellis_island script on your system so that we can avoid the duplications and potential other issues.

**Comment by knighttime** [comment]
@andkov 

The problem you are having with the duplicates is an issue with incorrectly merged data.
My fault - I solved the problem on my end but did not upload the new ellis_island script to github.
Will do that tomorrow

the package you need for reshape() to work is probably library(reshape2) : you should now be able to replicate my script on your computer. let me know if that wasn't it.

**Comment by knighttime** [comment]
7e86e07 #

**Comment by knighttime** [comment]
7e86e077cfd49adf11cf6bae1d38e12c17239abf #
